CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The default value for `draw_bold_text_with_bright_colors` is now `false`
 - Mirror OSC query terminators instead of always using BEL
 - Increased Beam, Underline, and Hollow Block cursors' line widths
-
+- Pixel scrolling now uses f64. Slow trackpad scrolling is not anymore ignored.
```

Comment:
It should be in a `Fixed` section. Also, you should avoid implementation details in a `CHANGELOG` entries, since we're not a library. Something like `Ignoring of slow touchpad scrolling` could be better.